src/HWMaps1.java

```diff
@@ -0,0 +1,16 @@
+import java.util.HashMap;
+import java.util.Map;
+//Write a Java program to test if a map contains a mapping for the specified key.
+public class HWMaps1 {
+    public static void main(String[] args) {
```

Comment:
Please do not create separate classes that are not needed. All exercises for this homework are best placed in one class